examples/netlifycms/src/containers/Post.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { withRouteData, Link } from 'react-static'
+import Moment from 'react-moment'
+//
+
+export default withRouteData(({ post }) => (
+  <div className="blog-post">
+    <Link to="/blog/">{'<'} Back</Link>
+    <br />
+    <h3>{post.data.title}</h3>
+    <Moment format="MMMM Do, YYYY">{post.data.date}</Moment>
+    <img className="image" src={require(`../images/uploads/${post.data.thumbnail}`)} alt="" />
```

Comment:
This is likely going to need to change to reference the `public` directory. Like this:
```javascript
<img className="image" src={`/images/uploads/${post.data.thumbnail}`)} alt="" />
```